docs/hyperscript.md

```diff
@@ -278,6 +278,29 @@ function doSomething(e) {
 m("div", {onclick: doSomething})
 ```
 
+Mithril adds event listeners to the DOM using [EventTarget#addEventListener](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener). This means you can provide a function (as above) or an [EventListener](https://developer.mozilla.org/en-US/docs/Web/API/EventListener) object. For example:
```

Comment:
[That's not how it works internally](https://github.com/MithrilJS/mithril.js/blob/next/render/render.js#L791-L832), and I'd rather not give people a false impression on that. You could still mention that both functions and event listener objects work, just don't include the implementation detail that gives people the wrong impression, especially of performance implications. (Changing existing listeners is a simple object property write, not a `removeEventListener` + `addEventListener` pair.)